mem0/llms/qwen.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+from typing import Dict, List, Optional
+
+from openai import OpenAI
+
+from mem0.llms.base import LLMBase
+from mem0.configs.llms.base import BaseLlmConfig
+
+class QwenLLM(LLMBase):
```

Comment:
You could create a unit test to cover this class.

Reply:
Sorry for replying so late. I have create the unit test.